docs/docs/gpt-researcher/tailored-research.md

```diff
@@ -79,4 +79,34 @@ if __name__ == "__main__":
 
     report = asyncio.run(get_report(query=query, report_type=report_type, report_source=report_source))
     print(report)
-```
\ No newline at end of file
+```
+
+### Research on LangChain Documents 🦜️🔗
+You can instruct the GPT Researcher to research on a list of langchain document instances.
+
+For example:
+
+```python
+from langchain_core.documents import Document
+from typing import List
+from gpt_researcher import GPTResearcher
+import asyncio
+
+
+async def get_report(query: str, report_type: str, report_source: str, documents: List[Document]) -> str:
+    researcher = GPTResearcher(query=query, report_type=report_type, report_source=report_source, documents=documents)
+    await researcher.conduct_research()
+    report = await researcher.write_report()
+    return report
+
+if __name__ == "__main__":
+    query = "What can you tell me about blue cheese based on my documents?"
+    report_type = "research_report"
+    report_source = "langchain_documents"
+
+    # using a LangChain retriever to get all the documents regarding cheese
+    # https://api.python.langchain.com/en/latest/retrievers/langchain_core.retrievers.BaseRetriever.html#langchain_core.retrievers.BaseRetriever.invoke
+    documents = langchain_retriever.invoke("All the documents about cheese")
```

Comment:
@hslee16 where is `langchain_retriever` initialized?

Reply:
@assafelovic I can't cover all the use case of all retrievers available to langchain - https://python.langchain.com/v0.1/docs/modules/data_connection/retrievers/

But I've added further example code in creating `PGVector` based retriever. 
As the main goal of this PR is to support langchain documents, I'm leaving the details regarding langchain retriever usage to the user.

Reply:
This is great thank you!